tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -143,7 +143,8 @@ def get_resource_owner_emails_from_resource(self, sqs_message, resource):
 
         # resolve the contact info from ldap
         non_email_ids = list(set(resource_owner_tag_values).difference(explicit_emails))
-        ldap_emails = [self.ldap_lookup.get_email_to_addrs_from_uid(uid) for uid in non_email_ids]
+        ldap_emails = list(chain.from_iterable([self.ldap_lookup.get_email_to_addrs_from_uid
```

Comment:
what is the traceback, and why does this fix it? a unit test would be nice.

Reply:
ah ic traceback is ref'd issue, given this was just added and passed review, its clear we need a unit test down this path.

Reply:
I've added a unit test to verify future regressions of this issue.